templates/telemetry/untrustedModules/untrustedModules.4.schema.json

```diff
@@ -69,6 +69,9 @@
           "type": "integer",
           "minimum": 0
         },
+        "xulLoadDurationMS": {
+          "type": "number"
```

Comment:
Sometimes it's useful to add `"minimum": 0`, but I'll leave that up to you.

Reply:
Makes sense here; I added these constraints in my latest push.

Regarding the version, is there any thing for me to do there?

Reply:
No, ops will handle that with a new deploy.